cli_reference/openshift_developer_cli/odo-release-notes.adoc

```diff
@@ -5,10 +5,95 @@ include::modules/common-attributes.adoc[]
 :context: odo-release-notes
 toc::[]
 
-include::modules/openshift-developer-cli-odo-notable-features.adoc[leveloffset=+1]
+[id="odo-notable-improvements_{context}"]
+== Notable improvements in {odo-title}{nbsp}{odo-ver}
 
-include::modules/openshift-developer-cli-odo-getting-support.adoc[leveloffset=+1]
+* IBM Z and PowerPC architecture binaries are now available.
+* Service catalog creation has improved to provide more useful output on service creation and deletion.
+* Information prompts have been added to component and URL creation, providing next step scenarios.
+* Log verbosity may now be provided using via environment variables: `export ODO_LOG_LEVEL=4; odo push`.
+* A push timeout may now be set `odo preference set PushTimeout 30`.
+* Overall documentation improvements. Now includes a description of {odo-title}'s inner architecture.
 
-include::modules/openshift-developer-cli-odo-technology-preview.adoc[leveloffset=+1]
+////
 
-include::modules/openshift-developer-cli-odo-known-issues.adoc[leveloffset=+1]
\ No newline at end of file
+= Notable features in the previous release
+
+{odo-title}{nbsp}{odo-ver} offers a simple way to develop applications on {product-title}. {odo-title} is completely client-based and requires no server within {product-title} cluster.
+
+{odo-title}{nbsp}{odo-ver} comes with:
+
+* Simple syntax and design centered around concepts familiar to developers such as projects, applications and components.
+* Compatibility with any language or runtime within the OpenShift catalog of component types.
+* Capability to add custom component types using custom image builders.
+* Official support for Java and Node.js component types.
+* Service catalog which allows users to connect applications deployed on {product-title} cluster to the Template Service Broker.
+* `odo service create` - an interactive mode which guides users through the process of creating applications.
+* `odo watch` - a feature that automatically detects changes to local code and applies the changes to the component in real time.
+
+////
+
+[id="odo-getting-support_{context}"]
+== Getting support
+
+.For Documentation
+
+If you have found an error or have suggestions for improving the documentation, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *{product-title}* product type and the *Documentation* component type.
+
+.For Product
+
+If you have found an error, encountered a bug or have suggestions for improving the functionality of {odo-title}, file an issue in link:http://bugzilla.redhat.com[Bugzilla]. Choose the *Red Hat odo for OpenShift Container Platform* product type.
+
+Provide as many details in the description of the issue as possible.
+
+[id="odo-fixed-known-issues_{context}"]
+== Fixed known issues
+
+* link:https://bugzilla.redhat.com/show_bug.cgi?id=1760573[Bug 1760573] An active pointer is not pointing to a current active project.
+
```

Comment:
The active pointer does not switch to the current active project after deletion of a project